### PR TITLE
feat(70325): add on screen keyboard cancel table --run-build

### PR DESCRIPTION
**Pull request by imam-auliya**
> [#70325] - FE POS - Done

Acceptance Criteria
- [x] Add keyboard on screen on cancel table

Changelog:  
- #70325 Add keyboard on screen on cancel table 

Figma Link: [figma](https://www.figma.com/design/O5YqNe7cyLghWFL1se2tAw/MakanYA!-POS-Desktop?node-id=7279-37275&t=hP6fQ0WUJTr0rANB-4)

image

### CODE REVIEW

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=48) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=48)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=48)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=48)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=48)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=48&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications: **Duplication**

File: src/app/modules/order/page/OrderPage.tsx [hunk 1]
```diff
@@ -730,14 +766,16 @@ const OrderPage: FC = () => {
               value={cancellNote}
               onChange={(event) => changeNote(event)}
               maxLength={200}
+              onFocus={() => setKeyboardVisibility(true)}
+              // onClick={() => setKeyboardVisibility(true)}
```
Comment (aldiz1711) [inline src/app/modules/order/page/OrderPage.tsx:770]: kalau ga dibutuhin bisa di takeout ya

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=48) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=48)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=48)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=48)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=48)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=48&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications: **Duplication**

**Review by aldiz1711 — APPROVED** [review]
OK

**Pull request merged**